src/cli.ts

```diff
@@ -81,6 +96,11 @@ Boolean values can be set to false using the 'no-' prefix.
       Prettier configuration
   --unreachableDefinitions
       Generates code for definitions that aren't referenced by the schema
+  --namePriority={${values(CliNameSource).join(',')}},...
```

Comment:
Could we keep this PR to `customName`, and avoid introducing a new `namePriority` option?